### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'dispose') 

### ISSUE THREAD

**Issue by SergeyArenzon** [google-map-react/google-map-react#1113]
After upgrading to react@18 my google maps crashing on render.

[https://ibb.co/bsGV76c](https://github.com/google-map-react/google-map-react/issues/url)
[https://ibb.co/ZGBhV8B](https://github.com/google-map-react/google-map-react/issues/url)
[https://ibb.co/X5vNTt5](https://github.com/google-map-react/google-map-react/issues/url)

OS: [mac]
Browser [any browser]

**Comment by lexcraw4d** [comment]
Any updates?!? I am also having the same issue....

**Comment by lemmen6** [comment]
same issue

**Comment by AtaydeEnrique** [comment]
Try removing <React.StrictMode></React.StrictMode> from your index.js file

**Comment by Mmojekwu** [comment]
Same issue

**Comment by fatihyildizhan** [comment]
Hi. Is there any way to fix it without removing `<React.StrictMode>` from `index.js` file? Thanks

**Comment by Michael-indochat** [comment]
Does anyone have the same problem in vite?

**Comment by TS22082** [comment]
> Does anyone have the same problem in vite?

I'm experiencing this with Vite right now.

**Comment by pyjun01** [comment]
Try removing Suspense & lazy loading

**Comment by Kirantechcs** [comment]
Yes, same issue here with Vite. We have to remove React.StrictyMode to make it work.

**Comment by lalkrishnaa** [comment]
> Hi. Is there any way to fix it without removing `<React.StrictMode>` from `index.js` file? Thanks

**Comment by ev3nst** [comment]
this repo is i think pretty much dead. i would recommend anyone stumble upon this to try --> https://github.com/JustFly1984/react-google-maps-api

**Comment by katherine95** [comment]
I had to change and use '@react-google-maps/api' which worked perfectly fine

The introduction of lazy loading and suspense to my project caused the issue to occur when changing routes and the current react component that has the GoogleMapReact is being destroyed.
